pubspec.yaml

```diff
@@ -6,18 +6,18 @@ version: 3.2.0
 homepage: https://github.com/lukef/qr.flutter
```

Comment:
you are supposed to upgrade the version number here too, either to 4.0.0
also update CHANGELOG.md